### DISCUSSION TITLE
What is the ideal directory structure?

### DISCUSSION THREAD

**Discussion by amlloyd** [gotson/komga#1295] [Q&A] [answered]
Let say have have a root directory `/comics`. I have a collection of Invincible comics, both individual issues as well as combined editions. What is the best directory structure to organize these?

**Comment by suineg** [comment, marked as answer]
/comics/Image/Invincible (2003)
/comics/Image/Invincible Special Battle Issue (2004)
/comics/Image/Invincible Special Spin-off (2005)
/comics/Image/Invincible (2025) <-- this is future vol 2 made up
/comics/Image/Invincible One-shots  <-- this is where I would put combined editions but it's probably super messy and I would say near unsolvable especially when we are working with a centralized database that assigns volumes the way it does

I used to be in the thought process that it should be more like

/comics/Image/Invincible/Invincible (2003) and the the rest inside of this

The problem is that you have too many team ups and overlaps and spinoffs, just gets gross.

**Reply by amlloyd** [reply]
I wish there was the ability to have nested folders.

**Comment by profzelonka** [comment]
This might not be popular but it still seems like Omnibus-way is the best way in terms of archiving comic collections and easily navigating comics in file explorer and Komga. For example:
/comics/Image/Invincible
/comics/Image/Invincible/2002 - Invincible Vol1 Omnibus (fan-made).cbz
/comics/Image/Invincible/2005 - Special Spinoff merged.cbz
/comics/Image/Invincible/2006 - Invincible meets Superman or whatever (One Shot TPB 2012).cbz 
/comics/Image/Invincible/2025 - Invincible Vol2 1-12 merged.cbz
/comics/Image/Invincible/2026 - Invincible Reborn (TPBs 1-3 merged).cbz

And if there's a lot of one shots, one could merged them together in release order only if it makes sense to do so, if they're in a similar year so they're not messy-ing up the directory.

The problem is you have to manually go thru each franchise to sort and merge to produce such logical archival folder and I don't see a program ever being smart enough to understand what will make sense logically in your folders. One could even throw in other similar franchises into the folder if for example Invincible gets other spin off titles not named Invincible. Komga would still just have it all in the same folder. The only point where this method doesn't work well in is scans that aren't great and will likely be replaced later - one would need to manually extract a merged and re-zip and rename to cbz.

I do something like that but in combination of new releases as single comics, all starting with year so they end up at the bottom, and once there's a lot or the series completes I can quickly merge them together into 1 for archival and ease of reading/sorting/storing.
